exchanges/subscription/store.go

```diff
@@ -207,20 +207,34 @@ func (s *Store) Len() int {
 	return len(s.m)
 }
 
-// PartitionByPresence returns two lists; one with the subscriptions that are present in the store and one with the
-// subscriptions that are not
-func (s *Store) PartitionByPresence(compare List) (matched, unmatched List) {
+// Contained returns a list of subscriptions in `compare` that are already in the store.
+func (s *Store) Contained(compare List) List {
 	if s == nil || s.m == nil {
-		return nil, compare // All are unmatched
+		return nil
 	}
 	s.mu.RLock()
 	defer s.mu.RUnlock()
+	var matched List
```

Comment:
Can move this to the signature if you want, and/but still use explicit return with it.